client/app/services/service-details/service-details.html

```diff
@@ -9,10 +9,10 @@
   </span>
   <div class="view-selector-bar">
     <div class="form-group toolbar-pf-view-selector">
-      <button class="btn btn-link usage-graphs-button" ng-click="vm.viewSelected('graphsView')"
-              ng-class="{active: vm.viewType === 'graphsView'}">
-        <i class="fa fa-tachometer"></i>
-      </button>
+      <!--<button class="btn btn-link usage-graphs-button" ng-click="vm.viewSelected('graphsView')"-->
```

Comment:
Why comment out VS. remove? If that feature is not something PM wants, shouldn't we pull all the codes?

Reply:
It was requested to be commented out rather than removed

Reply:
The request was to comment out rather than pull the functionality due to the fact that it cannot be supported as requested.  These were the items in the initial documentation 
- utilization trends aren't supported yet
- timeline of resources aren't supported yet

Reply:
That makes sense.  However, I'm not a fan of leave commented code in the repo (that's why we use Git so we can pull things in and out).  It is also a little confusing because in one part of the code, it is being commented out, and in another it's being deleted.  Ultimately, from an engineering perspective, if we aren't using the code...it needs to come out.  When it gets revisited, then we can restore it from the repo and/or start over.

Reply:
That's definitely your call, but as soon as the API support exists, we will want it back in the UI.

Reply:
gonna pull commented code